README.md

```diff
@@ -95,6 +95,13 @@ export VAULT_ID=opendatapolicing-hackathon/host_vars/localhost
 ansible-playbook  -i /usr/local/src/opendatapolicing-ansible/inventories/opendatapolicing-hackathon/hosts  /usr/local/src/opendatapolicing/ansible/project_token_setup.yml --ask-vault-pass -e "ocp_cluster_user_user=username" -e "ocp_cluster_user_password=changeme" 
 ```
 
+**Set Environment Variables**
+
+**ODP_ANSIBLE_INVENTORY** Defines location of git repo holding inventories and vaults. Default /usr/local/src/opendatapolicing-ansible
```

Comment:
is the `default` set in a file outside of this PR or is that a suggested default?`